Source/FountainSharp/FountainSharp.Parse/FountainParser.fs

```diff
@@ -26,6 +26,9 @@ let printDebug fmt par =
 [<Literal>]
 let EmptyLine = ""
 
+[<Literal>]
+let NewLine = "\n"
```

Comment:
It think Environment.NewLine is "\r\n". I had problems with that.
Maybe it can be usable, but then I have to review other parts as well.